File: kotlin/step/math/main_1712.kt
import kotlin.math.ceil

fun main() {
    val (a,b,c) = readLine()!!.split(" ").map { it.toInt() }
    if (c - b <= 0)
        print(-1)
    else{
        val ans = ceil(a.toDouble() / (c - b)).toInt()
        if (ans * (c-b) == a)
            print(ans + 1)
        else
            print(ans)
    }
}

File: cpp/step/math/main_1712.cpp
#include <iostream>
#include <cmath>

using namespace std;

int main() {
    int a, b, c;
    cin >> a >> b >> c;
    if (c - b <= 0)
        cout << -1 << endl;
    else {
        int ans = ceil(static_cast<double>(a) / (c - b));
        if (ans * (c - b) == a)
            cout << ans + 1 << endl;
        else
            cout << ans << endl;
    }
}